harpoon/features/stub.feature

```diff
@@ -0,0 +1,22 @@
+@isolated
+Feature: stub
+  Scenario Outline: templated stub
+    Given there is a stub
+    When a user updates the stub key "<key>" to "<value>"
+    Then the stub should have "<key>" equal "<value>"
+
+    Examples:
+        | key       | value |
+        | stub      | true  |
+
+  @isolated
+  Scenario: isolated stub
+    And there is a stub
+    When a user updates the stub key "foo" to "bar"
+    Then the stub should have "foo" equal "bar"
+
+  @skip:stub
+  Scenario: skipped stub
```

Comment:
So, unfortunately that's not really something we control without quite a bit of additional housekeeping. The filtering of tests by tag that we're relying on in `godog` just normally don't get printed. I could add an initial pass that grabs all features/scenarios without the filtering, then applies the filtering and diffs the two to get the skipped set, but that seems like it might be kinda down the road if we want to keep building on this.